support/build/extensions/no-debug-non-zts-20121212/tideways-4.0.3

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+# Build Path: /app/.heroku/php
+
+OUT_PREFIX=$1
+
+# fail hard
+set -o pipefail
+# fail harder
+set -eu
+
+source $(dirname $BASH_SOURCE)/../../_util/include/manifest.sh
+
+ZEND_MODULE_API_VERSION=$(basename $(dirname $0))
+ZEND_MODULE_API_VERSION=${ZEND_MODULE_API_VERSION#no-debug-non-zts-}
+
+case ${ZEND_MODULE_API_VERSION} in
+    20121212)
+        series=5.5
+        ;;
+    20131226)
+        series=5.6
+        ;;
+    20151012)
+        series=7.0
+        ;;
+    *)
+        echo "Unsupported PHP/Zend Module API version: ${ZEND_MODULE_API_VERSION}"
+        exit 1
+        ;;
+esac
+
+ext_dir=${OUT_PREFIX}/lib/php/extensions/no-debug-non-zts-${ZEND_MODULE_API_VERSION}
+bin_dir=${OUT_PREFIX}/bin
+
+dep_formula=${0#$WORKSPACE_DIR/}
+dep_name=$(basename $BASH_SOURCE)
+dep_version=${VERSION}
```

Comment:
No, it's set by the parent script. Look at the `newrelic` extension. All the logic is in that base script and the specific version file just sets the version and then sources the base script. The base script could of course also set the version automatically by looking at the `$0` file name, then the specific version script won't have to set the variable and can simply source the bar script.